tardis/energy_input/tests/test_util.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+import astropy.units as u
+import numpy.testing as npt
+import numpy as np
+
+import tardis.energy_input.util as util
+from tardis import constants as const
+
+# this is terrible practice
+R_ELECTRON = 2.8179403227e-15
```

Comment:
Maybe put these in a central location and import them whenever needed? It might make sense to have constants.py for all of TARDIS?